DiamondLightSource/hyperion
# #818 fix fgs plan system tests

Starts but doesn't finish #818 
Requires https://gitlab.diamond.ac.uk/controls/support/bl03s-builder/-/merge_requests/5 and https://github.com/DiamondLightSource/dodal/pull/371

While we're at it, this would be good too:
https://gitlab.diamond.ac.uk/controls/python3/s03_utils/-/merge_requests/14

### To test:
1. Build S03 with updated BL03S-BUILDER (note, you will need to change the install location in the BL03S-BUILDER `Dockerfile` if you aren't building it in `/dls_sw/work/${EPICS_VERSION}/support/BL03S-BUILDER`)
3. Confirm system tests in `test_fgs_plan.py` pass
